src/models/item.py

```diff
@@ -41,6 +41,10 @@ def __init__(self, data: dict):
                       10**price_including_taxes.get("decimals", 0))
         self.price = f"{self.price:.2f}"
         self.currency = item.get("price_including_taxes", {}).get("code", "-")
+        self.item_logo = item.get("logo_picture", {}).get(
+            "current_url", "https://tgtg-mkt-cms-prod.s3.eu-west-1.amazonaws.com/13512/TGTG_Icon_White_Cirle_1988x1988px_RGB.png")
+        self.item_cover = item.get("cover_picture", {}).get(
+            "current_url", "https://images.tgtg.ninja/standard_images/GENERAL/other1.jpg")
```

Comment:
```suggestion
        self.item_logo = item.get("logo_picture", {}).get(
            "current_url",
            "https://tgtg-mkt-cms-prod.s3.eu-west-1.amazonaws.com/"
            "13512/TGTG_Icon_White_Cirle_1988x1988px_RGB.png")
        self.item_cover = item.get("cover_picture", {}).get(
            "current_url",
            "https://images.tgtg.ninja/standard_images/GENERAL/other1.jpg")
```
Fixing E501 line too long (> 79 characters)